Sources/Fullscreen/SoldView/SoldView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public struct SoldViewModel {
+    let image: UIImage
+    let text: String
+    let button: String
+
+    public init(image: UIImage, text: String, button: String) {
+        self.image = image
+        self.text = text
+        self.button = button
+    }
+}
+
+public protocol SoldViewDelegate: NSObjectProtocol {
+    func soldView(_ soldView: SoldView, didSelectCallToAction: Button)
+}
+
+public class SoldView: UIView {
+    private lazy var imageView: UIImageView = {
+        let imageView = UIImageView(frame: .zero)
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.tintColor = .primaryBlue
+        return imageView
+    }()
+
+    private lazy var title: Label = {
+        let label = Label(style: .title1)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private lazy var button: Button = {
+        let button = Button(style: .callToAction)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    private lazy var stack: UIStackView = {
+        let stack = UIStackView(arrangedSubviews: [
+                imageView,
+                title,
+                button
+            ])
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.spacing = 24
```

Comment:
Should there be added an option to send a multiplier to the Spacing class? I cannot find a spacing option of the 24pt.

Reply:
That would be cool! I don't remember doing multipliers, actually I don't even know how to use them. Feel free to add support for it and show us how it's used :)